ci/scripts/driver.sh

```diff
@@ -17,65 +18,46 @@
 #################################################################
 # TODO using static build for GitHub CLI until fixed in HPC-Stack
 #################################################################
-GH=/home/Terry.McGuinness/bin/gh
-repo_url=${repo_url:-"https://github.com/NOAA-EMC/global-workflow.git"}
+export GH=${HOME}/bin/gh
 
 ################################################################
 # Setup the reletive paths to scripts and PS4 for better logging 
 ################################################################
-WF_ROOT_DIR="$(cd "$(dirname  "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd )"
+HOMEGFS_DIR="$(cd "$(dirname  "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd )"
```

Comment:
```suggestion
HOMEgfs="$(cd "$(dirname  "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd )"
```
There is no need to call the same thing with different names.